docs/docs/genai/prompt-version-mgmt/optimize-prompts/index.mdx

```diff
@@ -262,3 +262,24 @@ result = mlflow.genai.optimize_prompt(
     objective=objective,
 )
 ```
+
+## Performance Benchmarks
```

Comment:
Do we want to move this right below `Optimization Overview`? I think this section is quite appealing to users :)

Reply:
I moved this slightly up, I think get-started should come first as Opik does (https://www.comet.com/docs/opik/agent_optimization/overview)